public/router.php

```diff
@@ -25,15 +25,21 @@
 	date_default_timezone_set('UTC');
 }
 
-define('BASE_DIR', dirname(__DIR__).'/');
+$baseDir = str_replace('\\', '/', dirname(dirname(__FILE__)).'/');
+if (!is_file($baseDir.'pufferpanel')) {
+    $baseUrl = dirname($_SERVER['PHP_SELF']);
```

Comment:
Yeah sure but the master url must always exists then and filled in. It is a possibility to.  
BUT you still need to know where your files are. If you can tell me how you want to do that. I build it. For now this is the best and easiest solution.